corehq/apps/users/views/mobile/users.py

```diff
@@ -1685,6 +1731,60 @@ def link_connectid_user(request, domain):
         return HttpResponse()
 
 
+@require_can_edit_commcare_users
+@location_safe
+def send_connectid_invite(request, domain, user_id):
+    # Currently same as what send_confirmation_sms does
+    user = CommCareUser.get_by_user_id(user_id, domain)
+    if user.is_account_confirmed or not user.is_commcare_user() or user.domain != domain:
+        messages.error(request, "The user is already confirmed or is not a mobile user")
+        return HttpResponse(status=400)
+
+    invite_code = encrypt_account_confirmation_info(user)
```

Comment:
Todo; I will need to coordinate with mobile to get a link that opens ConnectID app with the invite intent.

Reply:
One other alternative is to send this as push notification (probably via the connectid server). That way the app is already handling it, and we know they are logged in. It will also allow us to hide all the info that doesn't matter to the user (like the code or username, which they won't be familiar with), since we can pass that in the data params.

Reply:
Yeah, definitely push notification would be better.

I remember Dave mentioning this specifically needed an SMS deeplink (I don't recall why). I will check with him and see if we can do push notification (on top of SMS).